cli/src/helpers/git.ts

```diff
@@ -112,9 +112,11 @@ export const initializeGit = async (projectDir: string) => {
 
     // --initial-branch flag was added in git v2.28.0
     const { major, minor } = getGitVersion();
-    if (major < 2 || minor < 28) {
+    if (major < 2 || (major == 2 && minor < 28)) {
       await execa("git", ["init"], { cwd: projectDir });
-      await execa("git", ["branch", "-m", branchName], { cwd: projectDir });
+      await execa("git", ["symbolic-ref", "HEAD", `refs/heads/${branchName}`], {
```

Comment:
maybe add a comment here. idk how commonly used it is, but i had never heard of the symbolic-ref command before

Reply:
Added a comment along with a link to the superuser answer I found about it.